src/sass/post-911-gib-status.scss

```diff
@@ -32,3 +32,28 @@ hr {
   	margin-top: 2em;
   }
 }
+
+.print-status {
```

Comment:
Awesome, thanks! If we find any issues in testing I will try that other approach.